src/server/server_family.cc

```diff
@@ -419,6 +423,28 @@ void SlowLog(CmdArgList args, ConnectionContext* cntx) {
   (*cntx)->SendError(UnknownSubCmd(sub_cmd, "SLOWLOG"), kSyntaxErrType);
 }
 
+void ValidateServerTlsFlags() {
```

Comment:
maybe worth adding what this function verifies. That we demand either password or a ca certificate as method for authentication when tls is enabled